test/integration/join-test.js

```diff
@@ -40,6 +40,7 @@ testRingpopCluster({
     size: 1
 }, 'one node can join', function t(bootRes, cluster, assert) {
     assert.ifErr(bootRes[cluster[0].hostPort].err, 'no error occurred');
+    assert.end();
```

Comment:
Note to reviewer: These tests assumed sync behavior which was fine until I wanted to use the `testRingpopCluster` framework async.